src/Microsoft.DotNet.Cli.Utils/Command.cs

```diff
@@ -93,7 +117,7 @@ private static string ResolveExecutableFromPath(string executable, ref string ar
             return executable;
         }
 
-        private static string ResolveExecutablePathFromProject(string executable, NuGetFramework framework)
+        private static string ResolveExecutablePathFromProject(string executable, NuGetFramework framework, ref CommandResolutionStrategy commandResolution)
```

Comment:
True. When I wrote that the previous code used ref and I did not want to add yet another output mechanism. :)